Doc/update (#351)
vtex-apps/product-summary: docs/ProductSummarySpecificationBadges.md

![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)

> ⚠️ 
>

> > The Product Specification Badges block has been deprecated in favor of the [Product Specifications](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-specifications) from the app `vtex.product-specifications`. Although support for this block is still granted, we strongly recommend you to update your store theme with the Product Speficiations's block in order to keep up with the component's evolution.

Reviewer: ```suggestion
> The Product Specification Badges block has been deprecated in favor of the [Product Specifications](https://developers.vtex.com/vtex-developer-docs/docs/vtex-product-specifications) app. Although support for this block is still available, we strongly recommend that you update your store theme with the Product Specifications app to stay up with the component's evolution.
```

## Customization

> In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-using-css-handles-for-store-customization).

Reviewer: ```suggestion
To apply CSS customizations in this and other blocks, follow the [Using CSS Handles for store customization](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-using-css-handles-for-store-customization) guide.
```
